### ISSUE TITLE
{weight=XX} in trigger excludes trigger from results

### ISSUE THREAD

**Issue by Lewwwk** [silentrob/superscript#194]
Adding a weight to a trigger seems to exclude the trigger altogether from consideration. 

Wonder if this might be related to #193

**Comment by silentrob** [comment]
Just to be clear, it should be `{weight=XX}` where XX is a value between 0 - 99

**Comment by Lewwwk** [comment]
Correct. That was how it was used. (updating title to reflect)

**Comment by silentrob** [comment]
So this is an old relic of RiveScript and I have considered dropping support for it, but it is good to know that it is still being used.

**Comment by Lewwwk** [comment]
Ya. It's useful in crafting a series of replies that change according to how  many times something has been asked. For example, it could be used in a catchall with the first reply being a longer description of expected behaviour, followed by gentle reminders, followed by silence of listing of currently available commands.

**Comment by mariusursache** [comment]
Michael, this sounds good. I’d like to add it to the Wiki docs, can you create an example small script that would illustrate your scenario? (I’m trying to think how weighting vs order of gambits in the topic can play together in an interesting useful way and an example would be relevant).

**Comment by Lewwwk** [comment]
This method is useful in creating more deterministic reply flows that can be used to help teach users to learn and fail gracefully.
```
// a catchall for any unknown single word commands
// weighted replies are used to help teach a user in sequence
+ *1
- {weight=90} I didn't catch that. Here are the commands I currently understand...
- {weight=80} Hmmm... did you mistype something?
- {weight=70} I didn't understand. Try again?
- {weight=60} Are you just trying to see what I will say?
// this could end like this so the bot no longer responds to that input 
// or could add a {keep} flagged reply to have a single reply that would for evermore be said to the user in response to that trigger.
```

**Comment by mariusursache** [comment]
I thought SuperScript only uses {weight} in triggers, not in replies? @silentrob?

**Comment by Lewwwk** [comment]
@mariusursache yes, my mistake. The original issue is correct, my mind just drifted to the reply case when you asked for an example. For trigger based weighting, I currently find that I have to manually locate the triggers (and be conscious of file naming) to have some control of processing order as weighting is currently not working in triggers for me.

**Comment by silentrob** [comment]
So there may be more we can do on the reply side, but this isn't a bug as the functionality does not exist today.

**Comment by Lewwwk** [comment]
We can't use {{weight=xx}} in a trigger? I know they are not supported in replies, but the original issue pertains to triggers.

**Comment by silentrob** [comment]
Why are you using ``{{weight=xx}}`` and not `{weight=xx}` ?

**Comment by Lewwwk** [comment]
oy veh - too tired - was using {weight=xx} as per original issue. editing my comment to correct.
